src/cockpit/config.py

```diff
@@ -27,6 +28,7 @@
 XDG_CONFIG_HOME = Path(os.getenv('XDG_CONFIG_HOME') or os.path.expanduser('~/.config'))
 DOT_CONFIG_COCKPIT = XDG_CONFIG_HOME / 'cockpit'
 
+IS_LITTLE_ENDIAN_MACHINE = sys.byteorder == 'little'
```

Comment:
TBH this doesn't feel like "configuration" to me, I'd rather just put that into dbus.py.